modules/videopress/videopress.php

```diff
@@ -22,7 +22,7 @@ public static function init() {
 
 	function __construct() {
 		$this->version = time(); // <s>ghost</s> cache busters!
-		add_action( 'jetpack_modules_loaded', array( $this, 'jetpack_modules_loaded' ) );
+		add_action( 'init', array( $this, 'jetpack_modules_loaded' ) );
```

Comment:
What do you think about renaming the called method as well not to confuse anyone?